packages/zoe/src/issuerTable.js

```diff
@@ -39,19 +39,26 @@ const makeIssuerTable = () => {
       const brandP = E(issuerP).getBrand();
       const brandIssuerMatchP = E(brandP).isMyIssuer(issuerP);
       const amountMathKindP = E(issuerP).getAmountMathKind();
-      /** @type {[Issuer,Brand,boolean,AmountMathKind]} */
-      const [
-        issuer,
-        brand,
-        brandIssuerMatch,
-        amountMathKind,
-      ] = await Promise.all([
-        issuerP,
-        brandP,
-        brandIssuerMatchP,
-        amountMathKindP,
-      ]);
-      // AWAIT /////
+
+      let issuer;
+      let brand;
+      let brandIssuerMatch;
+      let amountMathKind;
+      try {
+        /** @type {[Issuer,Brand,boolean,AmountMathKind]} */
+        ([issuer, brand, brandIssuerMatch, amountMathKind] = await Promise.all([
+          issuerP,
+          brandP,
+          brandIssuerMatchP,
+          amountMathKindP,
+        ]));
+        // AWAIT /////
+      } catch (err) {
+        console.error(err);
+        assert.fail(
```

Comment:
Please add a TODO here to rethrow the error after adding the details as a note (for upcoming assert library). The problem, which you can't fix well until then, is that the error you're throwing no longer has the informative stack trace of the error you caught.